generator/views/glean_ping_view.py

```diff
@@ -54,6 +54,8 @@ def to_lookml(self, bq_client, v1_name: Optional[str]) -> Dict[str, Any]:
         against the view in the explore.
         """
         lookml = super().to_lookml(bq_client, v1_name)
+        # ignore nexted join views
```

Comment:
Typo: nexted -> nested. It would also probably be a good idea to expand on this comment (e.g. with the additional context below)